homeassistant/components/rabbitair/config_flow.py

```diff
@@ -0,0 +1,152 @@
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Rabbit Air."""
+
+    VERSION = 1
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry: config_entries.ConfigEntry) -> OptionsFlow:
+        """Get the options flow for the Rabbit Air component."""
+        return OptionsFlow(config_entry)
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user", data_schema=STEP_USER_DATA_SCHEMA
+            )
+
+        errors = {}
+
+        try:
+            info = await validate_input(self.hass, user_input)
+        except CannotConnect:
+            errors["base"] = "cannot_connect"
+        except InvalidAccessToken:
+            errors["base"] = "invalid_access_token"
+        except InvalidHost:
+            errors["base"] = "invalid_host"
+        except TimeoutConnect:
+            errors["base"] = "timeout_connect"
+        except Exception as err:  # pylint: disable=broad-except
+            _LOGGER.debug("Unexpected exception: %s", err)
+            errors["base"] = "unknown"
+        else:
+            await self.async_set_unique_id(dr.format_mac(info["mac"]))
+            self._abort_if_unique_id_configured()
+            return self.async_create_entry(title=info["title"], data=user_input)
+
+        return self.async_show_form(
+            step_id="user", data_schema=STEP_USER_DATA_SCHEMA, errors=errors
+        )
+
+
+class CannotConnect(HomeAssistantError):
+    """Error to indicate we cannot connect."""
+
+
+class InvalidAccessToken(HomeAssistantError):
+    """Error to indicate the access token is not valid."""
+
+
+class InvalidHost(HomeAssistantError):
+    """Error to indicate the host is not valid."""
+
+
+class TimeoutConnect(HomeAssistantError):
+    """Error to indicate the connection attempt is timed out."""
+
+
+class OptionsFlow(config_entries.OptionsFlow):
+    """Handle an options flow for Rabbit Air."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Manage the options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        scan_interval: int = self.config_entry.options.get(
+            CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL
+        )
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Optional(
+                        CONF_SCAN_INTERVAL,
+                        default=scan_interval,
+                    ): vol.All(vol.Coerce(int), vol.Range(min=2)),
```

Comment:
Scan interval should not be part of integration configuration.
If someone wants a custom polling interval or schema, an automation that calls the `homeassistant.entity_update` service instead.

This mean this option flow can be fully removed, as this was the only option.

Reply:
Ok, removed it.